Include/internal/pycore_bytesobject.h

```diff
@@ -33,6 +33,44 @@ _PyBytes_ReverseFind(const char *haystack, Py_ssize_t len_haystack,
                      const char *needle, Py_ssize_t len_needle,
                      Py_ssize_t offset);
 
+#include <string.h>
+
+/** Helper function to implement the inplace repeat method on a buffer
+ *
+ * This method repeately doubles the number of bytes copied to reduce to the number of invocations of memcpy
+ */
+static inline void _PyBytes_RepeatInPlace_helper(char* buffer, Py_ssize_t start_len, Py_ssize_t end_len)
```

Comment:
These line lengths are creeping too long -- we usually try to keep things to 79 characters per line. See [PEP 7](https://peps.python.org/pep-0007/)

Reply:
I don't think we need this _helper function -- it's fine to call `memcpy(..., ..., 1);`